model_clone/mixins/clone.py

```diff
@@ -387,42 +386,70 @@ def __duplicate_o2o_fields(self, duplicate):
 
         return duplicate
 
-    def __duplicate_o2m_m2o_fields(self, duplicate):
-        """Duplicate many to one or one to many fields.
+    def __duplicate_o2m_fields(self, duplicate):
+        """Duplicate one to many fields.
 
         :param duplicate: The transient instance that should be duplicated.
         :type duplicate: `django.db.models.Model`
-        :return: The duplicate instance with all the many to one or one to many fields duplicated.
+        :return: The duplicate instance with all the one to many fields duplicated.
         """
         fields = set()
 
-        if self._clone_m2o_or_o2m_fields or self._clone_excluded_m2o_or_o2m_fields:
-            for f in self._meta.related_objects:
+        for f in self._meta.related_objects:
+            if f.one_to_many:
                 if any(
                     [
-                        any([f.many_to_one, f.one_to_many])
-                        and f.name in self._clone_m2o_or_o2m_fields,
-                        any([f.many_to_one, f.one_to_many])
-                        and f.name not in self._clone_excluded_m2o_or_o2m_fields,
+                        f.get_accessor_name() in self._clone_m2o_or_o2m_fields,
+                        self._clone_excluded_m2o_or_o2m_fields
+                        and f.get_accessor_name()
+                        not in self._clone_excluded_m2o_or_o2m_fields,
                     ]
                 ):
                     fields.add(f)
 
-        # Clone one to many/many to one fields
+        # Clone one to many fields
         for field in fields:
-            items = []
-            for item in getattr(self, field.related_name).all():
```

Comment:
@jackton1 The reason I removed `related_name` in test case is here: without specifying a `related_name`, `field.related_name` would is `None`, and then `getattr(self, field.related_name)` will raise `TypeError: getattr(): attribute name must be string`.

Reply:
Noted. Thanks @yuekui 

Released: https://github.com/tj-django/django-clone/releases/tag/v2.5.0